docs/soft-bindings.mdx

```diff
@@ -0,0 +1,19 @@
+---
+id: sb-algs
+title: Watermarking and fingerprinting algorithms
+hide_table_of_contents: true
+---
+
+[_Durable Content Credentials_](https://contentauthenticity.org/blog/durable-content-credentials) is a concept that helps content provenance to persist across content platforms by also using one or both:
+
+- **Undetectable watermarks**, actively inserted into the content
+- **Content fingerprinting**, passively computed from the content.
+
+Platforms might remove C2PA metadata if they don't yet support the standard or if they use automated process to, for example, provide multiple resolutions of an image which they then serve over the Web. By storing a copy of the metadata in an online database, you can use a watermark or a fingerprint to find it again.
+Combining both watermark and fingerprints further improves the robustness of the provenance information.
+
+In the C2PA specification, watermarks and content fingerprinting are known as [soft bindings](https://c2pa.org/specifications/specifications/2.1/specs/C2PA_Specification.html#_soft_bindings), which can be used to find digital content, even if the underlying bits differ. The C2PA specification requires that soft bindings be generated using one of the [algorithms](https://c2pa.org/specifications/specifications/2.1/specs/C2PA_Specification.html#_soft_binding_algorithm_list) approved by the C2PA Technical Working Group. The following table lists the algorithms from the [C2PA Soft Binding Algorithm List](https://github.com/c2pa-org/softbinding-algorithm-list).
+
+import JSONToTable from '@site/src/components/JSONToTable';
```

Comment:
We should NOT be duplicating that list on the CAI site - just point folks to the list, as above.